ddtrace/internal/core/crashtracking.py

```diff
@@ -1,68 +1,160 @@
-from typing import Callable
+import os
+import platform
+from typing import Dict
+from typing import Optional
 
 from ddtrace import config
 from ddtrace import version
-from ddtrace.internal.datadog.profiling import crashtracker
+from ddtrace.internal import forksafe
+from ddtrace.internal.compat import ensure_text
 from ddtrace.internal.runtime import get_runtime_id
-from ddtrace.internal.runtime import on_runtime_id_change
 from ddtrace.settings._agent import config as agent_config
 from ddtrace.settings.crashtracker import config as crashtracker_config
 from ddtrace.settings.profiling import config as profiling_config
 from ddtrace.settings.profiling import config_str
 
 
-is_available: bool = crashtracker.is_available
-failure_msg: str = crashtracker.failure_msg
-is_started: Callable[[], bool] = crashtracker.is_started
+is_available = True
+try:
+    from ddtrace.internal.native._native import CrashtrackerConfiguration
+    from ddtrace.internal.native._native import CrashtrackerMetadata
+    from ddtrace.internal.native._native import CrashtrackerReceiverConfig
+    from ddtrace.internal.native._native import CrashtrackerStatus
+    from ddtrace.internal.native._native import StacktraceCollection
+    from ddtrace.internal.native._native import crashtracker_init
+    from ddtrace.internal.native._native import crashtracker_on_fork
+    from ddtrace.internal.native._native import crashtracker_status
+except ImportError:
+    is_available = False
 
 
-@on_runtime_id_change
-def _update_runtime_id(runtime_id: str) -> None:
-    crashtracker.set_runtime_id(runtime_id)
+def _get_tags(additional_tags: Optional[Dict[str, str]]) -> Dict[str, str]:
+    tags = {
+        "language": "python",
+        "runtime": "CPython",
```

Comment:
```suggestion
        "runtime": sys.implementation.name,
```

? 🤷🏻 

(not important, I think we hardcode this in other places)

Reply:
```Python
>>> import sys
>>> sys.implementation.name
'cpython'
```
🤷 I'll just leave it as is

Reply:
okie dokie